CHANGELOG.md

```diff
@@ -1,3 +1,58 @@
+
+<!-- 8.2.0-beta2 START -->
+
+# 8.2.0-beta2 (2021-09-30)
+
+### Features and enhancements
+
+- **AccessControl:** Document new permissions restricting data source access. [#39091](https://github.com/grafana/grafana/pull/39091), [@gamab](https://github.com/gamab)
+- Add fiscal years and search to time picker. [#39073](https://github.com/grafana/grafana/pull/39073), [@oscarkilhed](https://github.com/oscarkilhed)
```

Comment:
What is the prefix for this feature or enhancement?

Reply:
```suggestion
- **TimePicker:** Add fiscal years and search to time picker. [#39073](https://github.com/grafana/grafana/pull/39073), [@oscarkilhed](https://github.com/oscarkilhed)
```

Reply:
@osg-grafana based on the PR it is a change to timepicker.